tests/CommitStatus.py

```diff
@@ -56,7 +56,7 @@ def testAttributes(self):
             datetime(2012, 9, 8, 11, 30, 56, tzinfo=timezone.utc),
         )
         self.assertEqual(self.statuses[0].creator.login, "jacquev6")
-        self.assertEqual(self.statuses[0].description, "Status successfuly created by PyGithub")
+        self.assertEqual(self.statuses[0].description, "Status successfully created by PyGithub")
```

Comment:
this change requires changing `tests/ReplayData/CommitStatus.testAttributes.txt` as well.